front/src/modules/trainschedule/components/ManageTrainSchedule/SimulationSettings.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import type { Distribution } from 'common/api/osrdEditoastApi';
+import SpeedLimitByTagSelector from 'common/SpeedLimitByTagSelector/SpeedLimitByTagSelector';
+
+import ConstraintDistributionSwitch from './ConstraintDistributionSwitch';
+import ElectricalProfiles from './ElectricalProfiles';
+
+type Props = {
+  selectedSpeedLimitByTag?: string;
+  speedLimitsByTags: string[];
+  dispatchUpdateSpeedLimitByTag: (newTag: string | null) => void;
+  constraintDistribution: Distribution;
+};
+
+export default function SimulationSettings({
```

Comment:
Same here, you can change `function` for `const`

Reply:
Done